pkg/operator/ceph/version/version.go

```diff
@@ -40,7 +40,7 @@ const (
 
 var (
 	// Minimum supported version
-	Minimum = CephVersion{18, 2, 0, 0, ""}
+	Minimum = CephVersion{19, 0, 0, 0, ""}
```

Comment:
Let's keep the GA version of Squid as the min version
```suggestion
	Minimum = CephVersion{19, 2, 0, 0, ""}
```